### PR TITLE
Added Titan pool to pools.json

### PR DESCRIPTION
**Pull request by jaybeddict1**
added "/Titan.io/" to "coinbase_tags" and associated address, 14hLEtxozmmih6Gg5xrGZLfx51bEMj21NW, to "payout_addresses"

https://twitter.com/OGBTC/status/1554845449290502144

### CODE REVIEW

File: pools.json [hunk 1]
```diff
@@ -1441,5 +1449,6 @@
         "tiger": "tiger",
         "1M1X": "1m1x",
         "OKMiner": "okminer"
+        "Titan": "titan"
```
Comment (nymkappa) [inline pools.json:1452]: Invalid JSON syntax, you're missing a comma

**Review by nymkappa — CHANGES_REQUESTED** [review]

File: pools.json (hunk 1 above)
Comment (jaybeddict1) [reply pools.json:1452]: thank you, corrected line in line 1451

**Review by nymkappa — APPROVED** [review]
Tested ACK @ 459e23e440d67c545d7638fde2a583ebad338d0f

```
Aug  9 22:06:12 [78533] DEBUG: <lightning> Pools.json sha | Current: 07ebb5292b95b2f92ae1fbfc473133874ff49abf | Github: 9e5171989664b15a9ab6016355118fbed12b887d
Aug  9 22:06:12 [78533] WARN: <lightning> Pools.json is outdated, fetch latest from https://raw.githubusercontent.com/jaybeddict1/mining-pools/patch-2/pools.json
Aug  9 22:06:12 [78533] DEBUG: <lightning> Found 139 unique mining pools
Aug  9 22:06:12 [78533] WARN: <lightning> No slug found for 'OKMINER', generating it => 'okminer'
Aug  9 22:06:12 [78533] DEBUG: <lightning> Add 'Titan' mining pool
Aug  9 22:06:12 [78533] WARN: <lightning> No slug found for 'ULTIMUS POOL', generating it => 'ultimuspool'
Aug  9 22:06:12 [78533] DEBUG: <lightning> Update pools table now
Aug  9 22:06:12 [78533] NOTICE: <lightning> Deleting blocks with unknown mining pool from height 130635 for future re-indexing
Aug  9 22:06:12 [78533] NOTICE: <lightning> Truncating hashrates for future re-indexing
Aug  9 22:06:12 [78533] INFO: <lightning> Mining pools.json import completed
Aug  9 22:06:12 [78533] NOTICE: <lightning> PoolsUpdater completed
```

Screen Shot 2022-08-09 at 10 08 51 PM

**Comment by nymkappa** [comment]
@jaybeddict1 Actually here too you can remove the `/` as it's not part of the coinbase tag, then we'll merge 👍🏻

**Pull request merged**